larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -67,17 +67,16 @@ typedef struct {
   // IDs are the arbitrary identifiers used by TF Lite to identify and access
   // memory buffers. They are unique in the entire TF Lite context.
   int im2col_id = kTensorNotAllocated;
-  int padding_buffer_id = kTensorNotAllocated;
-  int bitpacked_weights_buffer_id = kTensorNotAllocated;
   // In node->temporaries there is a list of tensor id's that are part
   // of this node in particular. The indices below are offsets into this array.
   // So in pseudo-code: `node->temporaries[index] = id;`
   int32_t im2col_index;
-  int32_t padding_buffer_index;
-  int32_t bitpacked_weights_buffer_index;
 
-  bool padding_cache_filled = false;
-  bool bitpacked_weights = false;
+  std::vector<std::uint8_t> padding_buffer;
+  std::vector<std::uint8_t> bitpacked_weights_buffer;
+
+  bool is_weight_bitpacked = false;
```

Comment:
these variables need to be right beside their corresponding buffer:
```
.. padding_buffer;
.. is_padding_correction_cached = false;

.. bitpacked_weights_buffer;
.. is_weight_bitpacked = false;
```
so someone who reads the code understands there is a related bool that needs to be updated after changing anything related to this buffer.